src/generated/rpg_saveinventory.h

```diff
@@ -49,7 +48,7 @@ namespace RPG {
 	inline bool operator==(const SaveInventory& l, const SaveInventory& r) {
 		return l.party_size == r.party_size
 		&& l.party == r.party
-		&& l.items_size == r.items_size
+		&& l.item_ids == r.item_ids
```

Comment:
fdelapena mentioned in chat that this line is duplicated. Have to admit that I never noticed this before in the other PRs, is this happening with all the "operator==" that have size fields?
Just a cosmetic issue, won't affect the result.

Reply:
Made an issue for this. Needs to be fixed in the comparison generation script.